exercises/practice/hello-world/hello-world.nix

```diff
@@ -0,0 +1,3 @@
+{
+  hello = {}: "Goodbye, Mars!";
```

Comment:
I'm not sure if we really want a function taking an empty set (and nothing else) here, or if we want instead just a constant value.

Reply:
I was under the impression that it has to be a function, not a variable. Now, looking at the Haskell, OCaml, Elm tracks, this doesn't appear to be the case (see the SML track for an example of the function approach). If it does need to be a function, maybe this would work better:
```nix
{ hello = _: "Goodbye, Mars!"; }
```

I was also trying to aim for consistency with the problem specification. The relevant test case is:
```json
    {
      "uuid": "af9ffe10-dc13-42d8-a742-e7bdafac449d",
      "description": "Say Hi!",
      "property": "hello",
      "input": {},
      "expected": "Hello, World!"
    }
```

If it were more idiomatic Nix for `hello` to be a constant value and not a function, I agree with changing it. We should maintain the same policy wherever there is empty input in the problem specification.

Reply:
It is true for Ruby as well, while the example code does not show it, there is no need to use a class or a module in Ruby to have the tests pass.

It is up to the track to determine how to work with the details of each exercise.